src/tools/plan-generator.ts

```diff
@@ -85,27 +100,16 @@ export class PlanGenerator {
 
   /** Generates a Kotlin `Plan.HandleConnection` from a HandleConnection. */
   async createHandleConnection(connection: HandleConnection): Promise<string> {
-    const schemaRegistry = [];
-    if (connection.type.hasVariable) {
-      schemaRegistry.push(await this.createSchemaForVariableHandleConnection(connection));
```

Comment:
For my own clarity: Instead of doing this, we are insisting on resolved type variables. Does that sound right?

Reply:
I want to double check: does this align with how we're creating protos?

Reply:
It's mostly that we have moved from creating the schema for type variables externally to `generateConnectionType` to now having the schema generation happen inside generateConnectionType whenever a concrete type cannot be found.

Reply:
It should align - the main point is that for handle connections we put in resolved types.

Reply:
Thanks!